docs/source/api/apollo-gateway.mdx

```diff
@@ -52,9 +60,14 @@ The core of the federated gateway implementation. For an example, see the ["impl
       },
     });
     ```
+
   * `introspectionHeaders`: `{ [key: string]: string }`
 
-    An object containing headers sent with introspection queries
+    An object containing HTTP headers to include in all introspection queries
```

Comment:
It's a little weird to call these "introspection," but I think the ship has sailed there.

Reply:
Agree. I do think this is deserving of a name that differentiates itself from an introspection query, since it's inaccurate to call it that (though it's very similar in concept).
`(?Partial) SDL query` maybe?